isaric/schemas.py

```diff
@@ -149,7 +148,6 @@ class Visit(BaseModel):
     treatment_monoclonal_antibody: bool | None = None
     treatment_other: list[str] = []
 
-    treatment_oxygen_mechanical_support: bool | None = None
```

Comment:
Same thing here .. There might be other ISARIC datasets where there will be matching fields / columns for 'oxygen_mechanical_support' ... 
We can remove the redundancy in the parsers, but keep the redundancy in the schemas - IMO..